ui-backend/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/util/EndpointUtil.java

```diff
@@ -308,6 +309,26 @@ public String getResponseWrapper(String responseType, Object response) {
 
   public Map<String, Result> getMetacardsWithTagByAttributes(
       String attributeName, Collection<String> attributeValues, Filter tagFilter) {
+    return getMetacardsWithTagByAttributes(attributeName, attributeValues, tagFilter, null);
+  }
+
+  public Map<String, Result> getMetacardsWithTagByAttributes(
+      String attributeName,
+      Collection<String> attributeValues,
+      String tag,
+      @Nullable Set<String> sourceIds) {
+    return getMetacardsWithTagByAttributes(
+        attributeName,
+        attributeValues,
+        filterBuilder.attribute(Core.METACARD_TAGS).is().like().text(tag),
```

Comment:
📝  Gonna be honest- I just added this in to make things easier to test because mocking filter builder is a pain